assets/javascripts/kitten/components/grid/side-layout.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import classNames from 'classnames'
+import bowser from 'bowser'
+import { SideGrid,
+         SideGridContent,
+         SideGridAside } from 'kitten/components/grid/side-grid'
+import { Grid, GridCol } from 'kitten/components/grid/grid'
+import DocLinkBox from 'kitten/components/box/doc-link-box'
+import { Title } from 'kitten/components/typography/title'
+
+export class SideLayout extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = { needsFullHeight: false }
+
+    this.renderDocLinkBox = this.renderDocLinkBox.bind(this)
+  }
+
+  needsFullHeight() {
+    return !!bowser.safari
+  }
+
+  componentDidMount() {
+    if (this.needsFullHeight()) {
+      this.setState({ needsFullHeight: true })
+    }
+  }
+
+  renderDocLinkBox(item) {
+    const { id, ...docLinkBoxProps } = item
+
+    return (
+      <div key={ id }
+           className={ classNames('k-u-margin-top-triple',
+                                  'k-u-margin-right-double',
+                                  'k-u-margin-left-quadruple') }>
+        <DocLinkBox { ...docLinkBoxProps } />
+      </div>
+    )
+  }
+
+  renderDocLinkBoxes() {
+    if (!this.props.sidebar) return
+
+    return this.props.sidebar.items.map(this.renderDocLinkBox)
+  }
+
+  render() {
+    const { className, ...sideGridProps } = this.props
+    const hasFluidLayout = (className || '').search('k-VerticalGrid__fluid') != -1
+    const hasFullHeightLayout = hasFluidLayout && this.state.needsFullHeight
+    const sideLayoutClassName = classNames(
+      className,
+      {
+        'k-VerticalGrid__fullHeight': hasFullHeightLayout,
+      }
+    )
+
+    return (
+      <SideGrid className={ sideLayoutClassName }
+                containerClassName="k-VerticalGrid__container"
+                rowClassName="k-VerticalGrid__element">
+        <SideGridContent>
+          <Grid>
+            <GridCol col-s="10" offset-s="1" col-l="8">
+              <Title modifier="secondary"
```

Comment:
Est-ce que le `Title` doit être dans ce composant ? J'aurais eu tendance à l'ajouter dans l'application et le passer par `this.props.children`.